app/controllers/podcast_metrics_controller.rb

```diff
@@ -97,22 +92,12 @@ def dropdays
         .order(first_rss_published_at: :desc)
         .paginate(params[:episode_dropdays], params[:per])
 
-    @interval = dropdays_params[:interval]
-    @fudged_range = if @interval == "DAY"
-      # first day is not a "complete" day
-      # fudged range matches chart selection
-      @dropday_range.to_i - 1
-    elsif @interval == "HOUR"
-      @dropday_range.to_i
-    end
     @dropdays = @episodes.map do |ep|
       if ep[:first_rss_published_at]
-        Rollups::HourlyDownload
-          .where(episode_id: ep[:guid], hour: (ep.first_rss_published_at..(ep.first_rss_published_at + @fudged_range.send(:"#{@interval.downcase}s"))))
-          .select(:episode_id, "DATE_TRUNC('#{@interval}', hour) AS hour", "SUM(count) AS count")
-          .group(:episode_id, "DATE_TRUNC('#{@interval}', hour) AS hour")
-          .order(Arel.sql("DATE_TRUNC('#{@interval}', hour) ASC"))
-          .load_async
+        date_start = ep.first_rss_published_at
+        date_end = ep.first_rss_published_at + (@dropday_range.to_i - 1).days
+
+        downloads_date_range_query(ep, date_start, date_end, "DAY")
```

Comment:
keeping dropdays as "DAY" interval for now and removed any options for otherwise.